sdk/samples/iot/pnp/pnp_thermostat_component.c

```diff
@@ -343,7 +337,7 @@ az_result pnp_thermostat_process_command_request(
     *out_status = AZ_IOT_STATUS_NOT_FOUND;
 
     IOT_SAMPLE_LOG_AZ_SPAN("Command not supported on Thermostat Sensor component:", command_name);
-    rc = AZ_ERROR_ITEM_NOT_FOUND; // Unsupported command or not this component's command
+    rc = AZ_ERROR_ITEM_NOT_FOUND; // Unsupported command
```

Comment:
Unrelated: Is this the correct `az_result` to set here?

Reply:
I am not certain what is most appropriate here.  What do you think?

Reply:
Yup it serves its purpose here.